service/internal/builders/receiver.go

```diff
@@ -1,31 +1,41 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package receiver // import "go.opentelemetry.io/collector/receiver"
+package builders // import "go.opentelemetry.io/collector/service/internal/builders"
```

Comment:
I've gone with a single package for all builders, but we could also have one package for each of them?

Reply:
Single package is fine for me